components/scg/src/scg-struct.js

```diff
@@ -62,38 +62,40 @@ function ScgFromScImpl(_sandbox, _editor, aMapping) {
 
                 let newMainNode = editor.scene.getObjectByScAddr(addr);
                 if (newMainNode) {
```

Comment:
Looks like issue that's not related to this pr, but should be fixed:
different thickness of sc-arc on edit mode and view mode
![image](https://github.com/ostis-ai/sc-web/assets/8840551/1fff6c42-ced2-42c2-960c-155ac341d9bb)